libs/template/labs/src/contentful/client/contentful-client.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Observable } from 'rxjs';
+
+export const ContentfulToken = 'oryx.ContentfulSpace';
+export const ContentfulSpace = 'oryx.ContentfulToken';
+
+declare global {
+  interface Environment {
+    readonly ORYX_CONTENTFUL_TOKEN?: string;
+    readonly ORYX_CONTENTFUL_SPACE?: string;
+  }
+}
+
+export interface ContentfulSearch {
+  content_type?: string;
+  query?: string;
+}
+
+export interface ContentfulResponse {
+  items: {
+    sys: {
+      contentType: { sys: { id: string } };
+    };
+    fields: Record<string, string>;
+  }[];
+}
+
+export type ContentfulResult = { contentful: Record<'name', string>[] };
+
+export interface ContentfulClientService {
+  getEntries(search: ContentfulSearch): Observable<ContentfulResponse>;
+}
```

Comment:
wondering why we have a contentful service. I thought we'd have a ContentService, with various ContentAdapters. 

So, 

Searchbox 
  -> Suggestions
    -> ContentService
       -> SAPI
       -> Contentful
       -> Stapi
       -> ...

Reply:
But it's client service. I just created it to reuse it inside  Suggestions and ContentService

Reply:
Maybe renaming to something like ContentfulApiService would make it more clear